lib/LoginStrategy.ts

```diff
@@ -251,23 +251,38 @@ export class LoginStrategy extends BaseStrategy {
   async offchainClaimsOf(did: string): Promise<OffchainClaim[]> {
     did = this.didUnification(did);
 
-    const transformClaim = (claim: OffchainClaim): OffchainClaim => {
+    const transformClaim = (
+      claim: OffchainClaim
+    ): OffchainClaim | undefined => {
       const transformedClaim = { ...claim };
-      const didFormatFields = ['iss', 'sub', 'subject', 'did'];
+      const didFormatFields = ['iss', 'sub', 'subject', 'did', 'signer'];
+
+      let invalidDIDProperty = false;
 
       Object.keys(transformedClaim).forEach((key) => {
         if (didFormatFields.includes(key)) {
-          transformedClaim[key] = this.didUnification(transformedClaim[key]);
+          const expectedEthrDID = transformedClaim[key];
```

Comment:
What this variable is for? Is there some reason why `transformedClaim[key]` can't be passed to `isValidErc1056`?

Reply:
Just to make code more descriptive about what `transformedClaim[key]` should be